tests/sps/validation/test_article_toc_sections.py

```diff
@@ -289,34 +454,42 @@ def test_validade_article_title_is_different_from_section_titles_returns_error(s
                 <front-stub>
                     <article-categories>
                         <subj-group subj-group-type="heading">
-                        <subject>Editorial</subject>
+                            <subject>Article</subject>
                         </subj-group>
                     </article-categories>
                     <title-group>
-                        <article-title>Editorial</article-title>
+                        <article-title>Article</article-title>
                     </title-group>
                 </front-stub>
-            </sub-article>        
+            </sub-article>
             </article>
             """
         )
         self.article_toc_sections = ArticleTocSectionsValidation(self.xmltree)
         expected = [
             {
-                'object': 'section title',
-                'article_title': {
-                    "es": "Editorial",
-                    "en": "Editorial",
-                },
-                'section_title': {
-                    "es": "Editorial",
-                    "en": "Editorial",
-                },
-                'result': False,
-                'message': 'ERROR: Article title ("Editorial") must not be the same'
-                           ' as the section title ("Editorial")'
+                'title': 'Article section title validation',
+                'xpath': ".//article-meta//subj-group[@subj-group-type='heading']/subject",
+                'validation_type': 'match',
+                'response': 'ERROR',
+                'expected_value': "'Artículo' (article title) different from 'Artículo' (section titles)",
+                'got_value': "article title: 'Artículo', section titles: 'Artículo'",
+                'message': 'article and section titles are the same',
+                'advice': 'Provide different titles between article and sections'
```

Comment:
Corrigir para "Provide different titles for article and section (`subj-group[@subj-group-type='heading']/subject`)'

Reply:
OK, @robertatakenaka